### PR TITLE
test: add cluster command end to end test

### LINKED ISSUE #137: Implement e2e tests for cluster command
We need to implement e2e tests for `solo cluster` command. The test suite should create its own cluster in `beforeAll()` assuming `kind` is available on the machine:

## TODO
- Create a new e2e test suite called: `test/e2e/commands/cluster.test.mjs` (remove other sequence number from other test files)
- We shall run it in its own cluster, e.g. `solo-test-cluster-cmd`
- Implement the e2e following structure as below:

```
describe('ClusterCommand` () => {
    beforeAll(async() => {
      await clusterCmd.Run('kind create cluster -n ${cluster-name})
    )
    
    afterAll(async() => {
      await clusterCmd.Run('kind delete cluster -n ${cluster-name})
    })

    it('solo cluster setup should fail with invalid cluster name', () => {
    })
    
    it('solo cluster setup should work with valid args', () => {
    })

    it('solo cluster reset should fail with invalid cluster name', () => {
    })
    
    it('solo cluster reset should work with valid args', () => {
    })
})
```
- Also please update `DEV.md` saying that `kind` is required for development.

### PR DESCRIPTION
**Pull request by JeffreyDallas**
## Description

This pull request changes the following:

* Adding an end-to-end test for cluster command 

### Related Issues

* Closes #137

### CODE REVIEW

**Comment by leninmehedy** [comment]
Let's split this PR and address in separate tickets as outlined here:

https://github.com/hashgraph/solo/issues/137
https://github.com/hashgraph/solo/issues/138

File: test/e2e/commands/cluster.test.mjs [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+
+  beforeEach(() => {
+    configManager.reset()
+  })
+
+  afterEach(async () => {
+    await sleep(5) // give a few ticks so that connections can close
+  })
+
+  it('should cleanup existing deployment', async () => {
```
Comment (jeromy-cannon) [inline test/e2e/commands/cluster.test.mjs:72]: there is no assertion on this test case

Comment (JeffreyDallas) [reply test/e2e/commands/cluster.test.mjs:72]: Updated

**Review by jeromy-cannon — APPROVED** [review]

**Comment by JeffreyDallas** [comment]
I made some mistake when trying to rebase, 
Opened a clean and new PR https://github.com/hashgraph/solo/pull/179

**Pull request closed without merging**